### ISSUE TITLE
datasets.IMDB.splits

### ISSUE THREAD

**Issue by AlexanderTrg** [pytorch/text#2153]
## ❓ Questions and Help

why am I getting different results in different cases (for example in jupyter and colab )?
after the same code 
`import torchtext
import torch
from torchtext.legacy import data
from torchtext.legacy import datasets

TEXT = data.Field()
LABEL = data.LabelField(dtype = torch.long)

train, test = datasets.IMDB.splits(TEXT, LABEL)
print(train[0].text)`

ps
i'd installed  before
!pip install torch==1.8.0 torchtext==0.9.0

jupyter: ['Powers', 'Booth', 'is', 'hypnotic'...]
colab : ['OK', 'heres', 'what', 'I', 'say:',...]

**Comment by achilatiao** [comment]
there is no legacy in current vision

**Comment by AlexanderTrg** [comment]
that's not the answer)
